server/models/Profile.js

```diff
@@ -4,7 +4,12 @@ const profileSchema = new mongoose.Schema({
   userId: {
     type: mongoose.Schema.Types.ObjectId,
     required: true,
-    ref: 'User'
+    ref: 'User',
+  },
+  accountType: {
+    type: String,
+    default: "",
```

Comment:
If we add a `pet_owner` default, we can further simplify things

Reply:
Done